### DISCUSSION TITLE
跨网段设备提示离线

### DISCUSSION THREAD

**Discussion by star2k** [al-one/hass-xiaomi-miot#1974] [Q&A]
![image](https://github.com/user-attachments/assets/e012da1b-c530-4def-9aee-147b6c69c055)

接入设备分布在单位不同网段，有些是固定内网IP，有些是在路由器之下

**Comment by al-one** [comment]
- https://github.com/al-one/hass-xiaomi-miot/issues/500#offline

**Reply by al-one** [reply]
部分设备是。
- https://python-miio.readthedocs.io/en/latest/troubleshooting.html#discover-devices-across-subnets
